inspirehep/modules/editor/api.py

```diff
@@ -42,6 +42,7 @@
     editor_use_api_permission,
```

Comment:
Cannot comment there, but you created a file called `%[` at the root of the repository.

Reply:
ops